lib/services/local/access_list.go

```diff
@@ -1029,3 +1043,18 @@ func (a *AccessListService) VerifyAccessListCreateLimit(ctx context.Context, tar
 	const limitReachedMessage = "cluster has reached its limit for creating access lists, please contact the cluster administrator"
 	return trace.AccessDenied(limitReachedMessage)
 }
+
+// conditionallyPreserveAWSIdentityCenterLabels preserves member labels if
+// it originated from AWS Identity Center plugin.
+// The Web UI does not preserve metadata labels so this function should be called
+// in every update/upsert member calls.
+func conditionallyPreserveAWSIdentityCenterLabels(old, new *accesslist.AccessListMember) {
```

Comment:
Nit: I'd rename to something shorter.
```suggestion
func keepAWSIdentityCenterLabels(old, new *accesslist.AccessListMember) {
```